e2e/app/base-page.ts

```diff
@@ -45,11 +45,14 @@ export default abstract class BasePage {
     return this.page.$eval(`${cssSelector}`, elem => elem.textContent.trim())
   }
 
-  async waitForNavigation() {
+  /**
+   * Click on element then wait for page navigation.
+   * @param {ElementHandle} clickElement
+   */
+  async clickAndWait(clickElement: ElementHandle) {
     return Promise.all([
-      this.page.waitForNavigation({waitUntil: 'load'}),
-      this.page.waitForNavigation({waitUntil: 'domcontentloaded'}),
-      this.page.waitForNavigation({waitUntil: 'networkidle0', timeout: 60000}),
+      this.page.waitForNavigation({waitUntil: ['domcontentloaded', 'networkidle0'], timeout: 90000}),
+      clickElement.click(),
```

Comment:
not 100% sure about the new method name. Welcoming suggestions for a better name.